src/twisted/web/test/test_template.py

```diff
@@ -217,7 +217,6 @@ def test_file(self) -> None:
 
 
 class XMLLoaderTestsMixin:
-
```

Comment:
Was this change also generated by auto-typing ?

Maybe a bug in auto-typing and upstream might want to know about it.

Reply:
autotyping runs via libcst which automatically applies black, this change is from that